modules/accessing-an-example-cluster-node-tuning-operator-specification.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id='accessing-an-example-node-tuning-operator-specification-{context}']
+= Accessing an example Node Tuning Operator specification
+
+Use this process to access an example Node Tuning Operator specification.
+
+.Procedure
+
+ . Refer to `/assets/tuned/07-cr-tuned.yaml` in the Operator's directory or to the
```

Comment:
I'm planning to rename `07-cr-tuned.yaml`, let's drop it and only keep `oc get Tuned/default -o yaml` as the example, that is not planned to be changed any time soon.